helper_yay.zsh

```diff
@@ -0,0 +1,85 @@
+#
+# yay aliases
+#
+
+#
+# General
+#
+
+# Print current yay configuration
+alias yayconf="yay -Pg"
+
+#
+# Install
+#
+
+#NOTE: Installing/upgrading individual packages is NOT supported. Sync and upgrade ALL on install.
+
+# install, sync, and upgrade packages
+alias yins="yay -Syu"
+
+# install packages without syncing
+alias yIns="yay -S"
+
+# install, sync, and upgrade packages (forcibly refresh package lists)
+alias yupg="yay -Syyu"
+
+# install packages by filename
+alias yinsf="yay -U"
+
+# install all packages in current directory
+alias yinsdir="yay -U *.pkg.*"
+
+
+#
+# Remove
+#
+
+# remove package and unneeded dependencies
+alias yr="yay -R"
+
+# remove package, unneeded dependencies, and configuration files
+alias yrm="yay -Rns"
+
+
+#
+# Query
+#
+
+# query package information from the remote repository
+alias yqry="yay -Si"
+
+# query package information from the local repository
+alias yQry="yay -Qi"
```

Comment:
Are most/all of these args identical to `pacman`?

A better question may be, which of these aliases and args is different from `pacman` and those in `init.zsh`?

Reply:
Yes they were, so I deleted them.